### PR TITLE
Update basics episode

### PR DESCRIPTION
**Pull request by bast**
Work in progress - do not merge yet. Is out here to get comments and to see what I am up to.

### CODE REVIEW

File: _episodes/02-basics.md [hunk 1]
```diff
@@ -54,6 +54,39 @@ $ git commit
...
+$ git config --global user.email yourname@example.com
+```
+
+The name and contact email will be recorded together with the code changes when we run `git commit`.
+
+It is convenient to set also the default text editor to use with Git. This you
+do with (**You can replace emacs with vim, nano or any other editor of your choice.**):
+
+```shell
+$ git config --global core.editor emacs
```
Comment (rkdarst) [inline _episodes/02-basics.md]: I've learned to always suggest nano by default (+ use it ourselves!)... unless there is some better option?

File: _episodes/02-basics.md [hunk 2]
```diff
@@ -54,6 +54,39 @@ $ git commit
...
+```
+
+Colors make it easier to inspect diffs and the staging area:
```
Comment (rkdarst) [inline _episodes/02-basics.md]: I think that colors are there by default these days?  Needs checking for versions.

File: _episodes/02-basics.md (hunk 1 above)
Comment (bast) [reply _episodes/02-basics.md]: I agree that nano is a better default. Patch coming up.

**Comment by bast** [comment]
Ready for review.

File: _episodes/02-basics.md (hunk 2 above)
Comment (bast) [reply _episodes/02-basics.md]: You are right. I will remove that section. It is default nowadays.

File: _episodes/02-basics.md [hunk 3]
```diff
@@ -151,9 +184,20 @@ Changes to be committed:
...
+Let us now commit the change to the repository:
+
+```shell
+$ git commit -m "adding ingredients and instructions"
+
+[master (root-commit) aa243ea] adding ingredients and instructions
+ 2 files changed, 10 insertions(+)
+ create mode 100644 ingredients.txt
+ create mode 100644 instructions.txt
+```
```
Comment (rkdarst) [inline _episodes/02-basics.md]: `git status` again after?  The first few times I am very pedantic about this.

**Comment by bast** [comment]
Thanks for review! Merging.

**Pull request merged**